File: transfer.js
const fs = require("fs");
const { resolve } = require("path");

let ticket = "";

fs.readFile("./resolved-tickets/2935.md", "utf8", async (err, data) => {
  if (err) throw new Error(err);
  ticket = data;
  console.log(ticket);
});

fs.mkdir("./converted", { recursive: true }, (err) => {
  if (err) return err;
});

console.log(ticket);

// fs.writeFile("./converted/2935.txt", ticket, (err) => {
//   if (err) return err;
// });

File: transfer.py
import sys
import os

ticket_num = sys.argv[1]
og_ticket_path = os.path.join("./resolved-tickets", "%s.md" % (ticket_num))

paths = [
    "./converted", 
    "./converted/text", 
    "./converted/json", 
    "./converted/tmp" 
]

for path in paths:
    os.mkdir(path)



with open(og_ticket_path, "r", encoding="utf-8-sig") as f:  
    lines = []
    for line in f:
        print(line, end="")
        # lines.append(f.readline())

print(lines)

# print(data)

# with open("%s.txt" % (ticket_num), 'w') as fp:




